client/api/editor/form.ts

```diff
@@ -19,7 +19,7 @@ export function getFormInstance(type: EDITOR_TYPE): IEditorAPI['form'] {
     }
 
     function getState() {
-        return planningApi.editor(type).manager.getState();
+        return planningApi.editor(type).manager?.getState?.();
```

Comment:
can optional chaining be avoided here?

Reply:
expose a method instead of check what you need to check in order to avoid optional chaining